content/docs/1.5.6/references/settings.md

```diff
@@ -399,6 +401,14 @@ The retained failed support bundle is for analysis purposes and needs to clean u
 
 Longhorn blocks support bundle creation when reaching the upper bound of the limitation. You can set this value to **0** to have Longhorn automatically purge all failed support bundles.
 
+#### Support Bundle Node Collection Timeout
+
+> Default: `30`
+
+The value in minutes specifies the timeout of node bundle collection for support bundle generation.
+
+When the timeout is reached, the support bundle generation will proceed without requiring the collection of node bundles.
```

Comment:
```suggestion
Number of minutes Longhorn allows for collection of node data for the support bundle.

If the collection process is not completed within the allotted time, Longhorn continues generating the support bundle without the uncollected node data.
```

Reply:
Questions:
- What type of data is collected for the "node bundle"? I'm asking because [Harvester](https://github.com/harvester/docs/pull/569/files) is more specific about the collection scope.
- Are there minimum and maximum values for this setting? What happens if the user specifies a value less than 30?

Reply:
> * What type of data is collected for the "node bundle"? I'm asking because [Harvester](https://github.com/harvester/docs/pull/569/files) is more specific about the collection scope.

The node agent currently collects the following, and there are also plans to include additional information and logs.
- host system info
- dmesg log
- host system log
- kubelet log

> * Are there minimum and maximum values for this setting? What happens if the user specifies a value less than 30?

The minimum value is 0, with no maximum limit. If the user specifies a value less than 30, the timeout will be set to that value in minutes.

Reply:
Thank you. Can we be more specific (say "node configuration and logs") in the description? This is to emphasize that we are not collecting user data.

*Number of minutes Longhorn allows for collection of node configuration and logs for the support bundle.*

*If the collection process is not completed within the allotted time, Longhorn continues generating the support bundle without the uncollected node data.*

Reply:
Thank you for the clarification.